src/lib.rs

```diff
@@ -1245,6 +1251,14 @@ impl Client {
         self.by(search.into())
     }
 
+    /// Find elements on the page.
+    pub fn find_all(
+        &self,
+        search: Locator,
+    ) -> impl Future<Item = Vec<Element>, Error = error::CmdError> + 'static {
+        self.by_all(search.into())
```

Comment:
Since nothing else needs `by_all`, I'd just place the contents of that method in here :)